tests/test_filtsmooth/test_gaussian/test_approx/_linearization_test_interface.py

```diff
@@ -52,9 +52,10 @@ def test_exactness_linear_model(self, rng):
         # Assert that the give the same outputs.
         received, info1 = linear_model.forward_rv(initrv, 0.0)
         expected, info2 = linearised_model.forward_rv(initrv, 0.0)
+
         crosscov1 = info1["crosscov"]
         crosscov2 = info2["crosscov"]
-        rtol, atol = 1e-10, 1e-10
+        rtol, atol = 1e-9, 1e-9
```

Comment:
the new-implementation-ukf matches the means and covs with relative error 1e-10, so I loosened this test a little.